src/js/facility-locator/reducers/facilities.js

```diff
@@ -1,5 +1,3 @@
-import { FETCH_VA_FACILITY, FETCH_VA_FACILITIES, SEARCH_FAILED } from '../actions';
```

Comment:
Are these constants not going to be used from the `reducers/index` module anymore? If not, might wanna remove the lines where they're defined.